tmpfiles.d/systemd.conf.in

```diff
@@ -45,9 +45,10 @@ a+ /run/log/journal/%m/*.journal* - - - - group:wheel:r--
 {% endif %}
 {% endif %}
 
-z /var/log/journal 2755 root systemd-journal - -
-z /var/log/journal/%m 2755 root systemd-journal - -
-z /var/log/journal/%m/system.journal 0640 root systemd-journal - -
+L+ /var/log/README     - - - -
```

Comment:
I'd prefer, if this was a separate .tmpfile.

Reply:
In the removed meson file, the file was conditionalized as `conf.get('HAVE_SYSV_COMPAT') == 1 and get_option('create-log-dirs')`. Shouldn't the same logic be used here?
```
conf.set10('CREATE_LOG_DIRS', get_option('create-log-dirs'))

{% if HAVE_SYSV_COMPAT and CREATE_LOG_DIRS %}
L+ /var/log/README  - - - -
{% endif %}
```

> I'd prefer, if this was a separate .tmpfile.

Is the 'create-log-dirs' option not enough?

Reply:
> > I'd prefer, if this was a separate .tmpfile.
> 
> Is the 'create-log-dirs' option not enough?

I guess that would be fine as well

Reply:
or a separate option for all those READMEs sprinkled across the system:
```
./var/log/README
./etc/init.d/README
./lib/modprobe.d/README
./lib/udev/hwdb.d/README
./lib/udev/rules.d/README
./usr/lib/tmpfiles.d/README
./usr/lib/sysusers.d/README
./usr/lib/sysctl.d/README
```

Reply:
> or a separate option for all those READMEs sprinkled across the system:

I think this is a completely separate subject. Those README files are in-place docs about a given directory.